packages/prisma/schema.prisma

```diff
@@ -315,12 +315,23 @@ model Team {
   apiKeys              ApiKey[]
   credentials          Credential[]
   accessCodes          AccessCode[]
+  isOrganization       Boolean                 @default(false)
+  organizationSettings OrganizationSettings?
   instantMeetingTokens InstantMeetingToken[]
   pendingPayment       Boolean                 @default(false)
 
   @@unique([slug, parentId])
```

Comment:
Notice there is no unique constraint [slug, isOrganization] which was there in original PR [here ](https://github.com/calcom/cal.com/pull/12828/files#diff-56fa4d384ba6fb94b130e9eb42c5cad00734b025ebd6c197f76873469725ae87R323)